modules/gateway-task/validation.tf

```diff
@@ -0,0 +1 @@
+
```

Comment:
Should we remove this file since it is empty?

Reply:
I've added a validation to ensure that only one of `var.retry_join_wan` or `var.enable_mesh_gateway_wan_federation` is provided.